pkg/controller/deployment/deployment_controller.go

```diff
@@ -65,14 +65,16 @@ type ReconcileDeployment struct {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileDeployment) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	ctx := context.Background()
```

Comment:
Isn't there a context in the request parameter?

Reply:
Unfortunately, no.